src/core/public/workspace/workspaces_client.ts

```diff
@@ -83,36 +154,54 @@ export class WorkspacesClient {
    * @param attributes
    * @returns
    */
-  public create = (
+  public async create(
     attributes: Omit<WorkspaceAttribute, 'id'>
-  ): Promise<IResponse<WorkspaceAttribute>> => {
+  ): Promise<IResponse<WorkspaceAttribute>> {
     if (!attributes) {
-      return Promise.reject(new Error('requires attributes'));
+      return {
```

Comment:
Do we still need this validation here if server side has defined the validation schema?